app/assets/stylesheets/application.scss

```diff
@@ -14,51 +14,78 @@
  @import "bootstrap-sprockets";
  @import "bootstrap";
 
-.field{
+
+.field {
   padding-bottom: 1em;
 }
 
-.container{
+.container {
   font-size: 18px;
 }
 
-.delete-button{
+.delete-button {
   color: red;
 }
 
-.page-links{
+.page-links {
   margin-top: 2em;
 }
 
-.logout-link{
+.logout-link {
   float: right;
 }
 
-.profile-link{
+.profile-link {
+  float: right;
+  padding-right: 1em;
+}
+
+.post-create-link {
   float: right;
   padding-right: 1em;
 }
 
-.top-links{
+.top-links {
   font-size: 24px;
 }
 
-.post-email{
+.post-username {
   padding-right: 1em;
 }
 
-.alert-notice{
+.alert-notice {
   color: green;
 }
 
-.alert-warning{
+.alert-warning {
   color: yellow;
 }
 
-.alert-alert{
+.alert-alert {
   color: red;
 }
 
-.user-profile-not-tweets{
+.post-feed-table {
+  a{
+    color: #333;
+    text-decoration: none;
+  }
+}
+.post-favorite {
+  padding-right: 1em;
+  a{
```

Comment:
Nitpicky things that I wouldn't expect you to have exposure to but these are guidelines that we're using for front end devs who are working with CSS:

- newline before a selector (in this case, a newline before `a` and a newline before `.post-favorite`)
- space between selector and `{` (after the `a` both here and below)

You've written really nice CSS here, these are nitpicky things we would usually catch with a linter. The reason we do this is to help readability and make it easier for devs to "pattern match" while scanning through the file.